FORMATTING_GUIDE.md

```diff
@@ -300,10 +300,10 @@ If you need to position an image or a code snippet within a list, use tabs to si
 **Example with images**
 
 ```
-1. To begin, select the rule in the Rule name column. The rule details pane opens.
+1. To begin, select the rule in the Rule name column. The rule details pane opens, as shown in the following image.
```

Comment:
Should "Rule name" be in bold as a UI element?

Reply:
Yes, updated.